### DISCUSSION TITLE
How to configure custom Esc for insert mode in Vim Mode for iPad?

### DISCUSSION THREAD

**Discussion by kamaz** [codesandbox/codesandbox-client#2270] [General]
# ❔ Question

I was wondering if there is any configuration option available ATM that would allow to configure custom mapping for Esc when *Vim Mode* is selected for iPad users? 

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | codesandbox.io
| Browser          | Safari/Chrome
| Operating System | iOS

**Comment by hastebrot** [comment]
The latest developer preview of iPadOS 13.4 includes settings to configure modifier keys of hardware keyboards.

![CEC94247-4F0E-4CE2-8E61-6047685E8BF0](https://user-images.githubusercontent.com/496255/76738044-29944a80-676a-11ea-938e-7ff9a6ab3d30.jpeg)

I wonder how to activate VIM mode on Safari/Chrome on iPad. The option is disabled in the CodeSandbox settings.
